### PR TITLE
Set unit npm script to --watch, added notes to readme

### PR DESCRIPTION
**Pull request by darintrimillos**
Makes unit testing better.

### CODE REVIEW

**Review by lancetharper — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ Runs [karma](https://karma-runner.github.io/1.0/index.html) unit tests. We're us
 
 Runs [Nightwatch](http://nightwatchjs.org/) end-to-end tests.
 
+`npm run unit`
+
+Runs unit tests. Keeps mocha active to watch for changes and re-runs tests. Use `it.only()` to watch just a single test.
```
Comment (KrisKnabel) [inline README.md:73]: How do you specify which single test?  Param to it.only()?  I think some additional details/instructions will be beneficial

Comment (darintrimillos) [reply README.md:73]: I'll add a better example

Comment (KrisKnabel) [reply README.md:73]: Does the watch function only run the test that was changed?  If so, that is really cool.

Comment (KrisKnabel) [reply README.md:73]: If you don't have to specify and run from the root that is.

**Review by lancetharper — APPROVED** [review]

**Pull request merged**